doc/api/readline.md

```diff
@@ -618,14 +618,14 @@ Using an `AbortController` to cancel a question.
 const ac = new AbortController();
 const signal = ac.signal;
```

Comment:
```suggestion
const signal = AbortSignal.timeout(10_000);
```

Reply:
The example description should also be changed in this case. The current description is "Using an `AbortController` to cancel a question."